app/auto-eligibility/db/models/customer.js

```diff
@@ -10,7 +10,9 @@ module.exports = (sequelize, DataTypes) => {
     },
     customer_name: DataTypes.STRING,
     business_name: DataTypes.STRING,
-    business_email: DataTypes.STRING,
+    business_email: {
+      type: DataTypes.CITEXT
```

Comment:
we can remove this now right and return back to string?

Reply:
done